apps/ledger-live-desktop/src/newArch/Collectibles/components/CollectionName.tsx

```diff
@@ -1,10 +1,11 @@
-import React, { memo, useMemo } from "react";
+import React, { memo } from "react";
 import { useNftCollectionMetadata } from "@ledgerhq/live-nft-react";
 import { Account, ProtoNFT } from "@ledgerhq/types-live";
 import NFTCollectionContextMenu from "~/renderer/components/ContextMenu/NFTCollectionContextMenu";
-import Skeleton from "~/renderer/components/Nft/Skeleton";
+import { Skeleton } from "LLD/Collectibles/components";
```

Comment:
what's the difference between `LLD` and `~` ?

Reply:
LLD is the alias for ~/newArch/